packages/patternfly-4/react-core/src/components/EmptyState/examples/SimpleEmptyState.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Title, Button, EmptyState, EmptyStateBody, EmptyStateAction } from '@patternfly/react-core';
+import { CubesIcon } from '@patternfly/react-icons';
+
+class SimpleEmptyState extends React.Component {
+  render() {
+    return (
+      <EmptyState>
+        <CubesIcon width="3em" height="3em" aria-hidden="true" />
```

Comment:
Also needs the pf-c-empty-state__icon class applied to it. Will it always be this icon or can the user pick any?

Reply:
@christiemolloy

Reply:
@jschuler the user can choose any icon that fits with their empty state pattern

Reply:
Thanks @jschuler.
I updated the code.